apps/storybook/src/Stacking.mdx

```diff
@@ -51,36 +51,38 @@ The stacking order is organised into six layers, from furthest (behind) to close
 The example below demonstrates the default stacking order. You can hover to test the stacking of the tooltip and crosshairs,
 and zoom in to test the stacking of the reset-zoom button in the floating toolbar.
 
-<Canvas>
-  <div style={{ display: 'flex', height: '25em' }}>
-    <VisCanvas
-      abscissaConfig={{ visDomain: [0, 41], showGrid: true }}
-      ordinateConfig={{ visDomain: [0, 20], showGrid: true }}
+{/* https://github.com/storybookjs/storybook/issues/19964 */}
+{/* <Canvas> */}
```

Comment:
Commenting this out for now since it causes an error. the JSX below still works; we just lose the ability to inspect the source code. I'll keep an eye on the closed issue linked above and open a new one if need be.

Reply:
Weird.